docs/src/pages/guides/deploy.md

```diff
@@ -31,6 +31,9 @@ By default, the build output will be placed at `dist/`. You may deploy this `dis
 
 ## GitHub Pages
 
+
+> By default, github pages will deny access to the `_astro/` directory. Add an empty `.nojekyll` file in the `public` folder for your site to function properly.
```

Comment:
can you rewrite this to make it clear that the deploy.sh script does this for you automatically, below?

(also, tests are failing for unrelated reason, should be fixed by rebasing/remerging onto main)